### PR TITLE
oasis prepares for war

### PR DESCRIPTION
**Pull request by cornercube**
## About The Pull Request

ports the merc heavy helmet into oasis, as the heavy steel helmet and the heavy steel breastplate (sprite pending)
has an amazing T6 protection vs bullets, but only T4 against melee, and still only T3 against lasers
a unique craftable that makes use of pre-war alloys to increase resistance vs high speed projectiles
due to low body coverage, it is poor against melee attacks
specifics: 150 bullet, 110 melee, 80 laser with 0.07 slowdown (0.91 with helmet, it is not named heavy for no reason)
compared to traditional metal armor mk2, 125 all-around defense with 0.15 slowdown (higher total values, more slowdown, less effective at bullet protection)

adds recipes for police pistol and police rifle, shopkeep and sheriff know it by default, a citizen loadout gets a book that teaches it to them

random ammo in the armory replaced with ammo the deputies can use

citizen description updated to mention you should not act outside of town interests
cowboy repeaters added to give to random citizens

## Why It's Good For The Game
gives oasis more unique gear
gives them very specialized anti-bullet gear
shame to watch all the fortune13 gear go to waste
also, a vain attempt to keep citizens from selling the town out or just allowing it to get trashed

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: heavy steel armor
add: recipes
add: a new book to teach the recipes

/:cl:

### CODE REVIEW

**Comment by Sinbest** [comment]
image

**Comment by cornercube** [comment]
> image

i just thought the helmet looked cool, and i wanted to steer gear more towards specialization

File: code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm [hunk 1]
```diff
@@ -384,8 +384,29 @@
 	subcategory = CAT_WEAPON
 	always_available = FALSE
 
+/datum/crafting_recipe/policerifle
+	name = "Police Rifle"
+	result = /obj/item/gun/ballistic/automatic/rangemaster
```
Comment (OneOneThreeEight) [inline code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm:389]: needs to be "/obj/item/gun/ballistic/automatic/marksman/policerifle"

**Review by OneOneThreeEight — CHANGES_REQUESTED** [review]
Changes may later require tweaking alongside the linearmor PR.

File: code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm (hunk 1 above)
Comment (cornercube) [reply code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm]: fixed

**Pull request merged**